aurora.ipynb

```diff
@@ -3,71 +3,88 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "a09fe8cd-1389-4d3d-8409-71c3dadc891e",
```

Comment:
I'm not an expert in jupyter notebooks, do you know what was the purpose of this ID field in? Is it just for some special feature if you need to reference one cell from another that we were anyways not using or something like this? It didn't seem to be used consistently (this one is a full uuid, but there is another below that was just a partial `b46c9eb7`...)

Reply:
No idea, but it was done by Jupyter automatically.